src/Raven.Server/Commercial/SetupWizard/ImportCertificateSetupUtils.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Server.Commercial.LetsEncrypt;
+
+namespace Raven.Server.Commercial.SetupWizard;
+
+public class ImportCertificateSetupUtils
```

Comment:
I wonder if `OwnCertificateSetupUtils` wouldn't be more accurate. This mode is more about providing own certificate while "import" might suggest it's about importing the certificate to already setup cluster. Also on UI we use "Provide your own certificate" (nothing about import).

Reply:
Changed